src/magical_chords/templates/library_layout.html

```diff
@@ -32,7 +32,7 @@ <h5>
             <a href="/song/?song_id={{ song.id }}" class="text-white-50">
               <h5 class="card-title" style="color:  rgb(255, 196, 0);">{{ song.title }} - {{ song.author }}</h5>
               <p class="card-text" style="color: rgba(255, 255, 255, 0.7);">
-                {{ song.song_content[:-100] }}
```

Comment:
it's ok to not do it here, but you will have to do it in the python code then...

Reply:
Yes, I removed, because it wasn't working, I suppose it is because SQLAlchemy is the language required to separate part of the content, instead of python/jinja